libc/CMakeLists.txt

```diff
@@ -641,7 +661,7 @@ add_enclave_library(
   ${MUSLSRC}/stdio/__stdio_write.c
   ${MUSLSRC}/stdio/stdout.c
   ${MUSLSRC}/stdio/__stdout_write.c
-  ${MUSLSRC}/stdio/__string_read.c
+  #${MUSLSRC}/stdio/__string_read.c
```

Comment:
Why this is commented out?

Reply:
Oh that file has been removed from musl. I'll delete the line :-)